microsoft-edge/progressive-web-apps-chromium/how-to/microsoft-store.md

```diff
@@ -126,19 +127,19 @@ The following figure shows the UI that is introduced when a user moves outside t
 
 ![The additional UI with URL and page title when the app is redirected to another domain](./microsoft-store-images/locale-redirection-additional-ui.png)
 
-### Domain redirection with browser-installed PWAs
+#### Domain redirection with browser-installed PWAs
 
 A Web App Manifest is tied to a single domain.  However, some PWAs use locale-specific domains for their customers in specific regions of the globe.  When visiting the PWA in a web browser, customers are seamlessly transitioned from the principal domain (for example, contoso.com) to a locale-specific domain (for example, contoso.co.ke), because the redirect happens during initial load of that website.
 
 Customers who install the PWA from Microsoft Edge would therefore install the PWA from the locale-specific domain.  Subsequent launches of the PWA go directly to that locale-specific domain, instead of first going to the principal domain.
 
-### Domain redirection with PWAs installed from the Microsoft Store
+#### Domain redirection with PWAs installed from the Microsoft Store
 
 PWAs that are installed from the Microsoft Store have a hard-coded start URL that is pointed at the principal domain.  When the PWA is launched, the PWA initially navigates to the principal domain, and then a customer may (as necessary) be redirected to their locale-specific domain. If that redirection occurs, the navigation is considered "out of scope".  As a result, the app displays the URL and page title at the top of the page.
 
 Displaying the URL and page title is a security feature to ensure that users know they have left the context of the PWA.  This added UI makes sense when a user loads a page from another website in the context of the PWA.  However, that added UI may be inappropriate when a user moves among domains that are all part of the same website.
 
-### Prevent the locale-specific URL and title from being displayed
+#### Prevent the locale-specific URL and title from being displayed
 
 To prevent the additional UI from being shown in a PWA that's installed from the Microsoft Store, you can use [URL Handlers](https://github.com/WICG/pwa-url-handler/blob/main/explainer.md) to enable the PWA to span multiple locale-specific domains.
```

Comment:
@LuHuangMSFT if I'm understanding this correctly, then that means removing the entire **Redirect to locale-specific domains without displaying additional UI** section. If we don't plan on supporting `url_handlers` much longer, then there really is no point in documenting how to use it to avoid the redirection UI in store-installed apps.

I proposed removing the entire section, and then adding something back in when we have a replacement available (`scope_extensions`?).

Do you agree?

Reply:
I think we can either remove this following block or the entire section:
```
This added UI makes sense when a user loads a page from another website in the context of the PWA.  However, that added UI may be inappropriate when a user moves among domains that are all part of the same website.

#### Prevent the locale-specific URL and title from being displayed

To prevent the additional UI from being shown in a PWA that's installed from the Microsoft Store, you can use [URL Handlers](https://github.com/WICG/pwa-url-handler/blob/main/explainer.md) to enable the PWA to span multiple locale-specific domains.
```

or

Reply:
Another option is to say that `scope_extensions` will be useful for this scenario but is not yet available.